modules/nixpkgs.nix

```diff
@@ -0,0 +1,137 @@
+# Licensed under GNU Lesser General Public License v3 or later, see COPYING.
+# Copyright (c) 2019 Alexander Sosedkin and other contributors, see AUTHORS.
+
+# Adapted from nixpkgs/home-manager.
```

Comment:
Given it's MIT-licensed, you might have to embed the original license header as described in
http://softwarefreedom.org/resources/2007/gpl-non-gpl-collaboration.html,
section 2.2 "Adding GPL'd modifications to permissive-licensed files".

Reply:
Is there a possibility to add just a small one-/two-liner like your gpl header and add the complete license add the repo root? Because I would rather not paste 20 or more useless comment lines as this should be done for some more files already present in nix-on-droid.. But I never really got in touch with these license things.